edx_sga/sga.py

```diff
@@ -2,58 +2,63 @@
 This block defines a Staff Graded Assignment.  Students are shown a rubric
 and invited to upload a file which is then graded by staff.
 """
+from __future__ import absolute_import
+
 import json
 import logging
 import mimetypes
 import os
-import urllib
-
 from contextlib import closing
 from zipfile import ZipFile
+
 import pkg_resources
-import pytz
+import six  # lint-amnesty, pylint: disable=import-error
+import six.moves.urllib.error  # lint-amnesty, pylint: disable=import-error
+import six.moves.urllib.parse  # lint-amnesty, pylint: disable=import-error
+import six.moves.urllib.request  # lint-amnesty, pylint: disable=import-error
 
-from lms.djangoapps.courseware.models import StudentModule  # lint-amnesty, pylint: disable=import-error
+import pytz
 from django.conf import settings  # lint-amnesty, pylint: disable=import-error
-from django.core.exceptions import PermissionDenied  # lint-amnesty, pylint: disable=import-error
-from django.core.files import File  # lint-amnesty, pylint: disable=import-error
-from django.core.files.storage import default_storage  # lint-amnesty, pylint: disable=import-error
-from django.template import Context, Template  # lint-amnesty, pylint: disable=import-error
+from django.core.exceptions import \
+    PermissionDenied  # lint-amnesty, pylint: disable=import-error
+from django.core.files import \
+    File  # lint-amnesty, pylint: disable=import-error
+from django.core.files.storage import \
+    default_storage  # lint-amnesty, pylint: disable=import-error
+from django.template import (  # lint-amnesty, pylint: disable=import-error
+    Context, Template)
```

Comment:
why we have such import changes?

Reply:
I think this is mostly due to running isort on the repo I just added import-error to disable list in pylintrc_tweaks, so this will no longer be necessary in the future.